Makefile

```diff
@@ -182,6 +182,9 @@ vendor: $(GO)
 	$(GO) mod tidy
 	$(GO) mod vendor
 
+auto-bumper:
+	$(GO) run $(shell ls tools/bumper/*.go |grep -v test) ${ARGS}
```

Comment:
consider please 2nd answer here
https://stackoverflow.com/questions/23695448/how-to-run-all-go-files-within-current-directory-through-the-command-line-mult

or this one `go run !(*_test).go` from there

Reply:
Done

Reply:
I think `!(*_test).go` does not play nice with Makefile.

Reply:
if its in a shell there or $$ as you said ?

Reply:
let's keep it simple. I prefer @qinqon 's suggestion.

Reply:
well if it works, is the same but syntactic sugar
up to you, curious if it can works with a simple chance in the Makefile and then we will know the new method
"|grep" -> "| grep" :)